File: generator-htt-angular/generators/app/templates/src/app/models/enumeration/enumeration.ts
export enum <%=enumeration.name%>{
  <% var i=0;
  for(value of enumeration.values) { %>
    <%=value.toUpperCase()%>="<%=value.toUpperCase()%>"<%if(i<enumeration.values.length-1){%>,
  <%} i++;} %>
}

File: generator-htt-springboot/generators/app/templates/enumeration/Enumeration.java
package <%=BasePackageName%>.enumeration;

public enum <%=enumeration.name%> {
    <% var i=0; for(value of enumeration.values) { %><%=value.toUpperCase()%>("<%=value.toUpperCase()%>")<%if(i<enumeration.values.length-1){%>,
    <%} i++;} %>;

    private String value;
    <%=enumeration.name%>(String value){
        this.value = value;
    }
    public String getValue(){
        return this.value;
    }
}
